avocado/utils/download.py

```
@@ -118,9 +119,10 @@ def url_download_interactive(url, output_file, title="", chunk_size=102400):
     """
     output_dir = os.path.dirname(output_file)
     with open(output_file, "w+b") as open_output_file:
-        with urlopen(url) as input_file:
+        with requests.get(url, stream=True) as response:
+            response.raise_for_status()
             try:
-                file_size = int(input_file.headers["Content-Length"])
+                file_size = int(response.headers.get('content-length', 0))
             except KeyError as exc:
                 raise ValueError("Could not find file size in HTTP headers") from exc
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)